### ISSUE TITLE
[BUG] GUI - unable to remove (peer) node

### ISSUE THREAD

**Issue by OverActiveBladderSystem** [Chia-Network/chia-blockchain#1047]
**Describe the bug**

unable to remove a node by clicking the "delete" trash can symbol, nor with "chia show -r nodeID"

**To Reproduce**

Not sure how to recreate the issue.

**Expected behavior**

I expect the node to be deleted from my listing, nodes above/below it can be deleted, but not the stuck node.

**Screenshots**

see attached screen shots of GUI, command line, and brief log of connections to IP.

**Desktop (please complete the following information):**

- OS: Windows 10 Pro 20H2 19042.804
- CPU: I9-9900k
- DRINK: Vitaminwater Orange-Orange

**Additional context**

![Screenshot (52)](https://user-images.githubusercontent.com/36348012/109084151-d55d4e80-76cc-11eb-8a11-262776a8732a.png)

![Screenshot (50)](https://user-images.githubusercontent.com/36348012/109082581-e2c50980-76c9-11eb-9e7d-031b8658d971.png)

![Screenshot (53)](https://user-images.githubusercontent.com/36348012/109083964-7ef01000-76cc-11eb-936d-d99c8b8e6565.png)

**Comment by OverActiveBladderSystem** [comment]
I also noticed, the stuck/bad peer counts as 1 of the 20 "target_outbound_peer_count: 20" so I have 19/20 good peers.

(tin foil hat) if reproduceable by external peers/influence, you could slowly cut nodes off from the network and thus farmers.

(hat taken off) minor annoyance, still farming, still winning blocks with 19/20 peers.

**Comment by OverActiveBladderSystem** [comment]
same peer has connected to me again, this time its fully sync'd with all the other peers, and I am able to delete the peer as expected normally (trash can delete), still unsure why I couldn't remove the peer before.

**Comment by mariano54** [comment]
Yes this is very strange behavior, thanks for the bug report. The expected behaviour is that the peer is removed instantly from the connections list.

**Comment by OverActiveBladderSystem** [comment]
> Yes this is very strange behavior, thanks for the bug report. The expected behaviour is that the peer is removed instantly from the connections list.

I seem to have come across another "ghosted" node I cannot delete, this time in RC3...

...the first edited log file, it appears I was communicating just fine with it, in the second log file, it appears thats where the disconnect happened and it got "ghosted" for lack of a better description, unable to remove it.

The timestamp of the last log entry for it seems to match the timestamp in the photo when I list all peers, after I tried to disconnect it.

![Screenshot (61)](https://user-images.githubusercontent.com/36348012/109400278-789ca680-790d-11eb-8539-05afb44e4a17.png)
![Screenshot (60)](https://user-images.githubusercontent.com/36348012/109400280-7b979700-790d-11eb-8c41-125a5fe34477.png)
![Screenshot (62)](https://user-images.githubusercontent.com/36348012/109400460-79820800-790e-11eb-8f68-d88583343357.png)
![Screenshot (63)](https://user-images.githubusercontent.com/36348012/109400723-ddf19700-790f-11eb-8283-2e29f25341ad.png)

**Comment by mariano54** [comment]
Potentially fixed by this:
https://github.com/Chia-Network/chia-blockchain/pull/1095

**Comment by mariano54** [comment]
I believe this is fixed in dev now. I have not seen it again. i'm closing this for now.

**Comment by OverActiveBladderSystem** [comment]
> I believe this is fixed in dev now. I have not seen it again. i'm closing this for now.

I recreated the issue again on RC5, I manually added "farmer1" as a peer today after some of us were tinkering around and checking our networks. I cannot remove the "farmer1" peer now. It might help diagnose things since "farmer1" is a chia run node? Either way, can't remove peer with trashcan, nor manually.

photo of command line attempt to remove peer, after GUI trashcan failed to.

![Screenshot (74)](https://user-images.githubusercontent.com/36348012/110258290-81255a80-7f67-11eb-8164-5dacdaeac30d.png)

photo of peer remaining after two removal methods attempted.

![Screenshot (73)](https://user-images.githubusercontent.com/36348012/110258296-884c6880-7f67-11eb-865c-945150de54b7.png)
